app/build.gradle

```diff
@@ -343,6 +348,9 @@ android {
 
             // set the app name
             resValue "string", "application_name", " CommCare LTS"
+            //setup action bar app name
+            buildConfigField "String", "APPLICATION_NAME", "\" CommCare LTS\""
```

Comment:
Is it possible to use the `application_name` defined above instead of creating a new field ?

Reply:
No it isn't because in ApplicationRecord.java class there is no context to get string.
But I did small code refactoring, I defined applicationName to use it for application_name, APPLICATION_NAME and  title_bar_name, this way is less chance to make mistake in some future changes.

Reply:
Unfortunately, I wasn't able to use BuildConfig fields in Manifest File. So the answer is no we cannot remove application_name.

Reply:
I see, I think for `ApplicationRecord` you should be able to use the application context (`CommCareApplication.instance()`) and hence able to use  `application_name` ?

Reply:
Yes it can use CommCareApplication.instance(), it is refactored now.